packages/cashier/src/components/withdraw-store-read-more-wrapper/withdraw-store-read-more-wrapper.tsx

```diff
@@ -0,0 +1,22 @@
+import { ReadMore } from '@deriv/components';
+import React, { ReactElement } from 'react';
+import { localize } from '@deriv/translations';
+
+type TWithdrawStoreReadMoreWrapperProps = {
+    error_content: string | ReactElement;
+    openDialog: () => void;
+};
+
+const WithdrawStoreReadMoreWrapper = ({ error_content, openDialog }: TWithdrawStoreReadMoreWrapperProps) => {
```

Comment:
@vinu-deriv Since you create this component inside the component folder and it has nothing withdrawal store or business related, could you make it more generic by changing the name, something like `ReadMoreWrapper`